packages/playwright/src/index.ts

```diff
@@ -119,6 +124,14 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     }, true);
   }, { scope: 'worker', timeout: 0 }],
 
+  _mockingProxy: [async ({ mockingProxy: mockingProxyOption, playwright }, use) => {
+    if (!mockingProxyOption)
+      return await use(undefined);
+    const localUtils: LocalUtils = (playwright as any)._connection.localUtils();
```

Comment:
Let's move all the code from this file into `src/client`, for example `Playwright._startMockingProxy()`. This way we'll avoid using multiple private apis.